ci/test/00_setup_env_native_fuzz.sh

```diff
@@ -15,3 +15,4 @@ export RUN_FUNCTIONAL_TESTS=false
 export RUN_FUZZ_TESTS=true
 export GOAL="install"
 export BITCOIN_CONFIG="--enable-fuzz --with-sanitizers=fuzzer,address,undefined CC=clang CXX=clang++"
+export CCACHE_SIZE=200M
```

Comment:
Effective size is now 114 MB:

https://travis-ci.org/github/bitcoin/bitcoin/jobs/708074351#L2863

Reply:
Still 120: https://travis-ci.org/github/bitcoin/bitcoin/jobs/712518048#L2866